google/cloud/internal/backoff_policy.h

```diff
@@ -126,11 +129,64 @@ class ExponentialBackoffPolicy : public BackoffPolicy {
   ExponentialBackoffPolicy(std::chrono::duration<Rep1, Period1> initial_delay,
                            std::chrono::duration<Rep2, Period2> maximum_delay,
                            double scaling)
-      : initial_delay_(initial_delay),
+      : ExponentialBackoffPolicy(initial_delay, initial_delay, maximum_delay,
+                                 scaling) {}
+
+  /**
+   * Constructor for an exponential backoff policy with an initial and minimum
+   * delay.
+   *
+   * Define the initial delay, minimum delay, maximum delay, and scaling factor
+   * for an instance of the policy. While the constructor accepts
+   * `std::chrono::duration` objects at any resolution, the data is kept
+   * internally in microseconds. Sub-microsecond delays seem unnecessarily
+   * precise for this application.
+   *
+   * @code
+   * using namespace std::chrono_literals; // C++14
+   * auto r1 = ExponentialBackoffPolicy<S,T>(0ms, 10ms, 500ms, 1.618);
+   * auto r2 = ExponentialBackoffPolicy<S,T>(0min, 10min, 10min + 2s, 1.002);
+   * @endcode
+   *
+   * @param initial_delay_range how long to wait after the first (unsuccessful)
+   *     operation.
+   * @param minimum_delay the minimum value for the delay between operations.
+   * @param maximum_delay the maximum value for the delay between operations.
+   * @param scaling how fast does the delay increase between iterations.
+   *
+   * @tparam Rep1 a placeholder to match the Rep tparam for @p
+   * initial_delay_range's type, the semantics of this template parameter are
+   * documented in `std::chrono::duration<>` (in brief, the underlying
+   * arithmetic type used to store the number of ticks), for our purposes it is
+   * simply a formal parameter.
```

Comment:
Nit: I know this is a copy of previous text, but it might be nice to eliminate the comma splices in both places while you're here.

Reply:
Done.